frontend/__tests__/src/fixtures.ts

```diff
@@ -38,7 +44,25 @@ export const BOARD_TYPES = {
   JIRA: 'Jira',
   LINEAR: 'Linear',
 }
+export const BOARD_FIELDS = ['Board', 'BoardId', 'Email', 'Project Key', 'Site', 'Token']
 
+export const MOCK_BOARD_URL = '/api/v1/kanban/verify'
+
+export const ERROR_MESSAGE = {
+  400: 'Jira verify failed: bad request',
+  404: 'Jira verify failed: 404 not found',
+  500: 'Jira verify failed: bad server',
```

Comment:
No bad server message, I think you mean `Internal server error`
No show 404 to the end user

Reply:
Because the backend is not written properly, the page test can only go to 404, so it needs to be processed